mne/beamformer/_dics.py

```diff
@@ -464,16 +595,17 @@ def apply_dics_csd(csd, filters, verbose=None):
     frequencies = [np.mean(dfreq) for dfreq in csd.frequencies]
     n_freqs = len(frequencies)
 
-    source_power = np.zeros((n_sources, len(csd.frequencies)))
+    source_power = np.zeros((n_sources * (3 * pick_ori),
```

Comment:
I don't think you have `pick_ori` in the `apply` functions.

Reply:
Yeah this was a bug in the implementation that got fixed in the other PR and needs to be ported over. This is probably causing many CI failures.